tests/gdx-tests-android/assets/data/maps/tiled/super-koalio/level1.tmx

```diff
@@ -16,7 +16,7 @@
   </data>
  </layer>
- <imagelayer name="koalio-color" x="352" y="136" opacity="0.5">
+ <imagelayer name="koalio-color" x="352" y="136" opacity="0.5" visible="0">
```

Comment:
These are needed for the ImageLayer test, maybe we can make that test use a copy of this map with the visible flags set to true.